src/main/java/com/android/volley/RequestQueue.java

```diff
@@ -131,15 +136,35 @@
      *
      * @param cache A Cache to use for persisting responses to disk
      * @param network A Network interface for performing HTTP requests
+     * @param cacheThreadPoolSize Number of cache dispatcher threads to create
      * @param threadPoolSize Number of network dispatcher threads to create
      * @param delivery A ResponseDelivery interface for posting responses and errors
      */
     public RequestQueue(
-            Cache cache, Network network, int threadPoolSize, ResponseDelivery delivery) {
+            Cache cache,
+            Network network,
+            int cacheThreadPoolSize,
```

Comment:
Same as the other comment: this will break things for a lot of people. Leave the old constructor with a call to this constructor with a cacheThreadPoolSize = 1.